usecases/objects/merge_test.go

```diff
@@ -429,9 +429,9 @@ func Test_MergeObject(t *testing.T) {
 				cls = tc.updated.Class
 			}
 			if tc.previous != nil {
-				if tc.previous.Properties != nil && tc.updated.Vector == nil {
-					m.modulesProvider.On("VectorizerName", mock.Anything).Return("some-module", nil)
-				}
+				// if tc.previous.Properties != nil && tc.updated.Vector == nil {
+				// 	m.modulesProvider.On("VectorizerName", mock.Anything).Return("some-module", nil)
+				// }
```

Comment:
I think that this part should be removed

Reply:
removed